src/v2/components/Fire.jsx

```diff
@@ -162,7 +162,7 @@ export default function Fire(props) {
 
     <div className="c7e-fire--caption">
       <p className="c7e-fire--location">
-        <strong className="c7e-fire--city-name">{cityName}</strong>
+        <strong className="c7e-fire--city-name">({index + 1}) {cityName}</strong>
```

Comment:
'index' is not stable and will change as new events arrive and old ones expire.  Should we really make the numbers so prominent?

Reply:
As I’ve touched upon previously, the fact that the list can change without the user being aware of that change is problematic. The best way to handle this is to ensure that the list is stable—at least until the user requests/acknowledges new detection events. But that’s not what we’re doing right now (and it may not even be feasible vis a vis pubsub) so `index` is definitely unstable.

We could simply not show the user a cardinal order as we did before this change. But obviously I would argue that omitting `index` is likely worse than showing an unstable value. Everything is still unstable, but without `index` there’s no simple, easily recalled data to help make sense of the relationship between detection events. 

In my testing, I observed that correlating tooltips/markers with list items involves a fair amount of cognitive overhead, especially when trying to make sense of a lot of potentially overlapping angles. It only gets worse when an angle’s origin includes multiple cameras.

One thing I find myself trying to do, for example, is differentiate proximate markers/polygons. This is not easy with either the v1 or v2 implementations. In v1, proximate objects are often split between multiple maps at different zoom levels. In v2, one map is panned and zoomed to accommodate the relevant objects. The presence of `index` mitigates these spatially confusing aspects by serving as a simple, easily recognized and recalled identifier. Place and camera names (e.g., Warner Springs · Sky Oaks -- SDSU Field Station) are less suitable because the information is not as immediately comprehended or called to memory—they’re just harder to use effectively.

One thing we could do instead of, or in addition to, using `index` is use a large array of say ~200 unique emojis/symbols such that even when `index` changes, we associate the same symbol with the same detection event. Having a large number of such symbols would allow us us to do that without repeating any one symbol until the list of ~100 detection events has been entirely replaced by newer events. Symbols like 🌶 🌽 🫐 🍅 🥝 would at least be quickly and easily differentiated even if they may seem odd at first or might come with unwanted connotations.

But another thing I find myself trying to do is mentally place a specific camera/angle in a cardinal order. I want to be able to tell, from the map, which markers are newer and which are older. Quickly gleaning the age of each detection event and anticipating how far back in time a click will take me seems important (i.e., to put each marker in context). It also allows users to know which time lapse to expect to see when they click to select a camera. Without `index`, this is only possible using the date and time (or worse place name, camera  name, date, and time). But remembering dates and times down to the second across hovers is daunting to say the least.

I guess there’s a chance that users by and large focus on such a small number of events that such issues are moot, but I don’t know….

Reply:
How about showing sourcePolygons.length and to disambiguate further (if needed) we could add '.<timeIndex>' where timeIndex is the nth element ordered by time with with the same sourcePolygons.length value?

Reply:
That might be a good tradeoff. Let me play around with it.

Reply:
Unfortunately, `sourcePolygons.length` is stable but its meaning isn’t clear. Nothing we render on the map right now really illustrates the `sourcePolygons.length` number nor how it might be used to orient yourself. In general, it’s a number without much context so we’d need to define that context somehow. Something like… “Overlapping fires: 1”. But even in simple cases, this number is more complicated than that—it’s more like… “Other potential fires overlapping this fire at the time it was detected: 1”. And the actual polygons drawn don’t really correspond to this number because we’re drawing all of the known overlapping fires (not just the one’s known at the time of this fire’s detection). 😝

Even if we are able to clarify the meaning of this number, it’s not really satisfying the need for a touchstone/anchor point because specific `sourcePolygons.length` values are shared—the most common case being `1` (i.e., no overlapping fires). Adding a `timeIndex` (“the nth element ordered by time with with the same `sourcePolygons.length`”) probably won’t shed any light either. For one thing, if I am interpreting it correctly, `timeIndex` is based on (a) the number of detection events with equal `sourcePolygons.length` values and (b) their time order within the larger list—and these may (will?) change when the list of detection events changes (i.e., `timeIndex` wouldn’t be much more stable than `index`).

After playing with `sourcePolygons.length`, I still think that—given the trade offs—`index` is preferable. Especially preferable to no anchor point at all.

However, if you feel strongly that it is the wrong direction to take/precedent to establish, I’m happy to remove this aspect from the PR. We can revisit the topic at a later date when we have more experience, and perhaps with more data.

Reply:
How about {cityName} ({sourcePolygons.length} 'th)  E.g., "Warm Springs (1st)"
I'm thinking the '(st|nd|th)' part may help clarify the role of the number, and demoting it towards the end addresses your point regarding it not being meaningful in majority of cases where the number stays at 1.

If you still don't like that, I agree with you that we should revisit at a later date.

Reply:
Adding an ordinal suffix (`th`, ...) helps a little (I was already demoting the number to follow the place name). It sort of differentiates `sourcePolygons.length` from `index`, but the user is still left guessing as to what the `sourcePolygons.length` number signifies (you probably have to be familiar with the code to know that lower means detected earlier). In the test/debug dataset, there are numerous camera locations with the same ordinal (e.g., 2nd) for multiple angles. Overall, I’m inclined to omit any such number.

![screencast 2022-08-07 11-23-57](https://user-images.githubusercontent.com/377413/183305612-0cbe0e25-5f59-47ad-8a03-cf97b0408d69.gif)